Description: adapte the audio of Sugon W330-H35A2
 Compared with the naming rules of hda sound card,
 this sound card has some special names, which
 need to be adapted here.
Author: Chengyi Zhao <zhaochengyi@uniontech.com>
Origin: https://gerrit.uniontech.com/plugins/gitiles/base/pulseaudio/+/4f7e97b63c6bde1f1b07aebd781c1feff50a77d2
Bug: https://pms.uniontech.com/zentao/bug-view-74346.html
Last-Update: 2022-05-25

--- pulseaudio-15.0+dfsg1.orig/src/modules/alsa/mixer/paths/analog-input-front-mic.conf
+++ pulseaudio-15.0+dfsg1/src/modules/alsa/mixer/paths/analog-input-front-mic.conf
@@ -24,6 +24,9 @@ description-key = analog-input-microphon
 [Jack Front Mic]
 required-any = any
 
+[Jack Front Mic - Input]
+required-any = any
+
 [Jack Front Mic Phantom]
 required-any = any
 state.plugged = unknown
--- pulseaudio-15.0+dfsg1.orig/src/modules/alsa/mixer/paths/analog-input-linein.conf
+++ pulseaudio-15.0+dfsg1/src/modules/alsa/mixer/paths/analog-input-linein.conf
@@ -23,6 +23,9 @@ priority = 81
 [Jack Line]
 required-any = any
 
+[Jack Line - Input]
+required-any = any
+
 [Jack Line Phantom]
 required-any = any
 state.plugged = unknown
--- pulseaudio-15.0+dfsg1.orig/src/modules/alsa/mixer/paths/analog-input-rear-mic.conf
+++ pulseaudio-15.0+dfsg1/src/modules/alsa/mixer/paths/analog-input-rear-mic.conf
@@ -24,6 +24,9 @@ description-key = analog-input-microphon
 [Jack Rear Mic]
 required-any = any
 
+[Jack Rear Mic - Input]
+required-any = any
+
 [Jack Rear Mic Phantom]
 required-any = any
 state.plugged = unknown
--- pulseaudio-15.0+dfsg1.orig/src/modules/alsa/mixer/paths/analog-output-headphones.conf
+++ pulseaudio-15.0+dfsg1/src/modules/alsa/mixer/paths/analog-output-headphones.conf
@@ -47,6 +47,9 @@ state.unplugged = unknown
 [Jack Headphone]
 required-any = any
 
+[Jack Headphone - Output]
+required-any = any
+
 [Jack Headphone Phantom]
 required-any = any
 state.plugged = unknown
--- pulseaudio-15.0+dfsg1.orig/src/modules/alsa/mixer/paths/analog-output-speaker.conf
+++ pulseaudio-15.0+dfsg1/src/modules/alsa/mixer/paths/analog-output-speaker.conf
@@ -72,6 +72,9 @@ state.unplugged = unknown
 [Jack Speaker - Output]
 required-any = any
 
+[Jack Desktop Speaker - Output]
+required-any = any
+
 [Element Hardware Master]
 switch = mute
 volume = merge
